hugegraph-core/src/main/java/org/apache/hugegraph/auth/StandardAuthManager.java

```diff
@@ -77,6 +76,10 @@ public class StandardAuthManager implements AuthManager {
     private final TokenGenerator tokenGenerator;
     private final long tokenExpire;
 
+    private List<String> ipWhiteList;
```

Comment:
mark final

Reply:
mark final maybe we cant change it anymore?

Reply:
> mark final maybe we cant change it anymore?

for `Object`, `final`/`static` will just fix the **Memory Point Address**, won't influence the **inner value**

Reply:
> > > mark final maybe we cant change it anymore?
> > 
> > 
> > for `Object`, `final`/`static` will just fix the **Memory Pointer Address**, won't influence the **inner value**
> > just like the **head pointer** in the linked list
> 
> like this？ we cant change object pointer. so we cant use `this.ipWhiteList = ipWhiteList` in setWhiteIPs func.
> 
> ```
> private final List<String> ipWhiteList;
> 
> public void setWhiteIPs(List<String> ipWhiteList) {
>     this.ipWhiteList.clear();
>     this.ipWhiteList.addAll(ipWhiteList);
> }
> ```